config/README.md

```diff
@@ -26,8 +26,20 @@ The config files contain a features object that can be used to determine whether
 }
 ```
 
+### Testing Feature Flags Locally
+
 If you want to temporarily enable/disable some feature flags for a given build, you can do so by setting the `ENABLE_FEATURES` and/or `DISABLE_FEATURES` environment variables. Set them to a comma separated list of features you want to enable/disable, respectively:
 
 ```bash
 ENABLE_FEATURES=manage/plugins/compatibility-warning DISABLE_FEATURES=code-splitting,reader npm start
 ```
+
+### Testing Feature Flags On Any Environment
+
+If you want to temporarily enable/disable some feature flags on a particular environment (calypso.live included) you can add a `?flags=` query parameter to the URL.
+
+- `?flags=foo` enables feature *foo*.
+- `?flags=-bar` disables feature *bar*.
+- `?flags=foo,-bar` enables feature *foo* and disables feature *bar*.
+
+E.g. https://wordpress.com/?flags=manage/plugins/compatibility-warning
```

Comment:
@Copons this is not quite accurate because this feature is not available (and should not be available) on horizon, production, wpcalypso.  Mentioning wordpress.com as the URL is also misleading because that only works if you're using that URL to access the staging environment.

Suggestion for revision:

```markdown
### Testing Feature Flags via URLs

If you want to temporarily enable/disable some feature flags you can add a `?flags=` query parameter to the URL.

**Note** that this **only** works on development, staging, and calypso.live, **not** in production (we don't want to allow public users to do this on `*.wordpress.com`).

- `?flags=foo` enables feature *foo*.
- `?flags=-bar` disables feature *bar*.
- `?flags=foo,-bar` enables feature *foo* and disables feature *bar*.

E.g. http://calypso.localhost:3000/?flags=manage/plugins/compatibility-warning
```

Reply:
@Copons are you planning to submit a follow-up PR to clarify how this feature works, or shall I?

Reply:
&rarr; #22818

Reply:
@nylen Sorry for not answering, I was AFK!
Thanks for the follow-up, and for pointing out that this doesn't work on prod (I was sure it did!). 🙇